pyproject.toml

```diff
@@ -83,7 +83,7 @@ exclude = [
 
 [tool.black]
 line-length = 120
-target-version = ["py311"]
+target-version = ["py38"]
```

Comment:
this is the only fix right? the rest is just formatting

Reply:
not just formatting, it was incompatible syntax.